pkg/detectors/apiflash/apiflash_test.go

```diff
@@ -1,121 +1,84 @@
-//go:build detectors
-// +build detectors
-
 package apiflash
 
 import (
 	"context"
 	"fmt"
 	"testing"
-	"time"
 
-	"github.com/kylelemons/godebug/pretty"
+	"github.com/google/go-cmp/cmp"
+
 	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/engine/ahocorasick"
+)
 
-	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
-	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+var (
+	validPattern = `
+	apiflash_key: a1b2c3d4e5f6g7h8i9j0k1l2m3n4o5p6
+	apiflash_url: abc123XYZ456def789ghijklm
```

Comment:
According to the [documentation](https://apiflash.com/documentation), this parameter should be an actual URL, not a plain text string limited from 21 to 30 characters. 

The current regex in use is: `urlPat: \b([a-zA-Z0-9\S]{21,30})\b`.

Reply:
Created a OSS JIRA ticket for this.